src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java

```diff
@@ -93,7 +93,7 @@ public String getBuildParams(SubBuild subBuild) {
                 } catch (Exception e) {
                     continue;
                 }
-                String value = stringParameter.value;
+                String value = (String) stringParameter.getValue();

```

Comment:
Hmm, would be useful to patch the core if it's still not fixed. Will cause FindBugs warnings, but does not matter so far

Reply:
The method is declared to return `Object` even though the actual value is a `String`. Could be improved in core but hardly seems a priority.